vkcsfm/data_source_vkcs_db_datastore_capabilities.go

```diff
@@ -0,0 +1,259 @@
+	AllowMajorUpgrade      types.Bool                             `tfsdk:"allow_major_upgrade"`
+	AllowUpgradeFromBackup types.Bool                             `tfsdk:"allow_upgrade_from_backup"`
+}
+
+type VkcsDBDatastoreCapabilityParamsModel struct {
+	Name         types.String  `tfsdk:"name"`
+	Required     types.Bool    `tfsdk:"required"`
+	Type         types.String  `tfsdk:"type"`
+	ElementType  types.String  `tfsdk:"element_type"`
+	EnumValues   types.List    `tfsdk:"enum_values"`
+	DefaultValue types.String  `tfsdk:"default_value"`
+	Min          types.Float64 `tfsdk:"min"`
+	Max          types.Float64 `tfsdk:"max"`
+	Regex        types.String  `tfsdk:"regex"`
+	Masked       types.Bool    `tfsdk:"masked"`
+}
+
+func (d *VkcsDBDatastoreCapabilities) Metadata(ctx context.Context, req datasource.MetadataRequest, resp *datasource.MetadataResponse) {
+	resp.TypeName = "vkcs_db_datastore_capabilities"
+}
+
+func (d *VkcsDBDatastoreCapabilities) Schema(ctx context.Context, req datasource.SchemaRequest, resp *datasource.SchemaResponse) {
+	resp.Schema = schema.Schema{
+		Attributes: map[string]schema.Attribute{
+			"id": schema.StringAttribute{
+				Computed:    true,
+				Description: "ID of the resource",
+			},
+
+			"region": schema.StringAttribute{
+				Computed:    true,
+				Optional:    true,
+				Description: "The `region` to fetch availability zones from, defaults to the provider's `region`.",
+			},
+
+			"datastore_name": schema.StringAttribute{
+				Required:    true,
+				Description: "Name of the data store.",
+			},
+
+			"datastore_version_id": schema.StringAttribute{
+				Required:    true,
+				Description: "ID of the version of the data store.",
+			},
+
+			"capabilities": schema.ListNestedAttribute{
+				Computed: true,
+				NestedObject: schema.NestedAttributeObject{
+					Attributes: map[string]schema.Attribute{
+						"name": schema.StringAttribute{
+							Computed:    true,
+							Description: "Name of data store capability.",
+						},
+						"description": schema.StringAttribute{
+							Computed:    true,
+							Description: "Description of data store capability.",
+						},
+						"params": schema.ListNestedAttribute{
+							Computed:     true,
+							NestedObject: dataSourceDatabaseDatastoreCapabilitiesParam(),
+						},
+						"should_be_on_master": schema.BoolAttribute{
+							Computed:    true,
+							Description: "This attribute indicates whether a capability applies only to the master node.",
+						},
+						"allow_major_upgrade": schema.BoolAttribute{
+							Computed:    true,
+							Description: "This attribute indicates whether a capability can be applied in the next major version of data store.",
+						},
+						"allow_upgrade_from_backup": schema.BoolAttribute{
+							Computed:    true,
+							Description: "This attribute indicates whether a capability can be applied to upgrade from backup.",
+						},
+					},
+				},
+				Description: "Versions of the datastore.",
```

Comment:
We should replace Versions with Capabilities here, it's already existed typo.

Reply:
Done